config/application.rb

```diff
@@ -46,5 +46,14 @@ class Application < Rails::Application
     config.to_prepare do
       Devise::Mailer.layout "mailer"
     end
+
+    Rails.application.configure do
+      config.lograge.formatter = Lograge::Formatters::Json.new
+      config.lograge.enabled = true
+
+      config.lograge.custom_options = lambda do |_event|
+        { name: 'RMT' }
```

Comment:
may be better to call this `managing-registers` to keep it consistent with the CF app name and repo name.